Dockerfile

```
@@ -1,46 +1,95 @@
+###
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
 
-ARG GOPHISH_VERSION="0.11.0-cisa.1"
+###
+# Unprivileged user setup variables
+###
 ARG CISA_UID=421
-ENV CISA_HOME="/home/cisa" \
-    SCRIPT_DIR="/usr/local/bin"
+ARG CISA_GID=${CISA_UID}
+ARG CISA_USER="cisa"
+ENV CISA_GROUP=${CISA_USER}
+ENV CISA_HOME="/home/${CISA_USER}"
+
+###
+# Upgrade the system
+###
+RUN apt-get update --quiet --quiet \
+    && apt-get upgrade --quiet --quiet
 
-RUN addgroup --system --gid ${CISA_UID} cisa \
-  && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
+###
+# Create unprivileged user
+###
+RUN groupadd --system --gid ${CISA_GID} ${CISA_GROUP} \
+    && useradd --system --uid ${CISA_UID} --gid ${CISA_GROUP} --comment "${CISA_USER} user" ${CISA_USER}
 
-RUN apt-get update && \
-apt-get install --no-install-recommends -y \
-unzip \
-ca-certificates \
-wget && \
-apt-get install -y sqlite3 libsqlite3-dev && \
-apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+###
+# Install everything we need
+#
+# Install dependencies are only needed for software installation and
+# will be removed at the end of the build process.
+###
+ENV DEPS \
+    ca-certificates \
+    libsqlite3-dev \
+    sqlite3
+ENV INSTALL_DEPS \
+    unzip \
+    wget
+RUN apt-get install --quiet --quiet --yes \
+    --no-install-recommends --no-install-suggests \
+    $DEPS $INSTALL_DEPS
 
 COPY bin/get-api-key ${SCRIPT_DIR}
 
-USER cisa
-WORKDIR ${CISA_HOME}
 # TODO: Revert from cisagov/gophish back to gophish/gophish after all of our
 # pull requests have been merged; including, but potentially not limited to:
 # - https://github.com/gophish/gophish/pull/1484
 # - https://github.com/gophish/gophish/pull/1486
 # See https://github.com/cisagov/gophish-docker/issues/25 for details.
-RUN wget -nv https://github.com/cisagov/gophish/releases/download/v${GOPHISH_VERSION}/gophish-v${GOPHISH_VERSION}-linux-64bit.zip && \
-unzip gophish-v${GOPHISH_VERSION}-linux-64bit.zip && \
-rm -f gophish-v${GOPHISH_VERSION}-linux-64bit.zip
+RUN wget --no-verbose https://github.com/cisagov/gophish/releases/download/v${GOPHISH_VERSION}/gophish-v${GOPHISH_VERSION}-linux-64bit.zip \
+    && unzip -d ${CISA_HOME} gophish-v${GOPHISH_VERSION}-linux-64bit.zip \
+    && rm --force gophish-v${GOPHISH_VERSION}-linux-64bit.zip
+RUN chmod +x ${CISA_HOME}/gophish \
+    && ln --symbolic --no-dereference --force /run/secrets/config.json ${CISA_HOME}/config.json \
+    && mkdir --parents ${CISA_HOME}/data \
+    && ln -snf ${CISA_HOME}/data/gophish.db ${CISA_HOME}/gophish.db
+
+###
+# Remove install dependencies
+###
+RUN apt-get remove --quiet --quiet $INSTALL_DEPS
 
-RUN chmod +x gophish && ln -snf /run/secrets/config.json config.json && \
-mkdir -p data && ln -snf data/gophish.db gophish.db
+###
+# Clean up aptitude cruft
+###
+RUN apt-get --quiet --quiet clean
+RUN rm -rf /var/lib/apt/lists/*
```

Since these are logically connected they should be one `RUN` command.

```suggestion
RUN apt-get --quiet --quiet clean \
    && rm -rf /var/lib/apt/lists/*
```